src/js/views/eva-variant-view.js

```diff
@@ -84,12 +84,12 @@ EvaVariantView.prototype = {
             }
         });
 
-        $('#variantViewTabs li').click(function (event) {
-            $(this).toggleClass("active");
-            $(this).siblings().removeClass("active");
-        });
+        // $('#variantViewTabs li').click(function (event) {
+        //     $(this).toggleClass("active");
+        //     $(this).siblings().removeClass("active");
+        // });
```

Comment:
Same as in the gene view file, why is this block commented out? Please remove if no longer necessary.